pkgs/development/libraries/libosmium/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, cmake, protozero, expat, zlib, bzip2, boost }:
+
+stdenv.mkDerivation rec {
+  name = "libosmium-${version}";
+  version = "2.14.2";
+
+  src = fetchFromGitHub {
+    owner = "osmcode";
+    repo = "libosmium";
+    rev = "v${version}";
+    sha256 = "123ri1l0a2b9fljgpwsl7z2w4i3kmgxz79d4ns9z4mwbp8sw0250";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ protozero zlib bzip2 expat boost ];
+
+
+  meta = with stdenv.lib; {
+    description = "Fast and flexible C++ library for working with OpenStreetMap data";
+    homepage = "https://osmcode.org/libosmium/";
+    licence = licences.boost;
```

Comment:
typo: should be "licen**s**es"

Reply:
Should I fix this in a new commit, or &mdash; as this was never a valid meta attributes set &mdash; rewrite the commit that introduced the typo?

Reply:
Feel free to squash it into the existing commit. I'd prefer we end up with just the four, one for each application and the maintainer addition. :)